templates/docs/patterns/navigation.md

```diff
@@ -51,16 +51,27 @@ To update to new class names:
 ### Sub-navigation
 
 Sub-navigation drop-down menus can be added to the navigation by including one
-or more `p-subnav` components.
+or more `p-navigation__item--dropdown-toggle` components.
 
-To open the subnav-menu you need to set the `is-active` class on `p-subnav` element, but also set `aria-hidden` to `false` on `p-subnav__items`.
+To open the subnav-menu you need to set the `is-active` class on `p-navigation__item--dropdown-toggle` element, but also set `aria-hidden` to `false` on `p-navigation__dropdown`.
 
-By default the sub-navigation menus are left-aligned with their parent, if you'd prefer the menu to be positioned from the right, use the `p-subnav__items--right` modifier. This can be seen in the "My account" menu in the example.
+By default the sub-navigation menus are left-aligned with their parent, if you'd prefer the menu to be positioned from the right, use the `p-navigation__dropdown--right` modifier. This can be seen in the "My account" menu in the example.
 
 <div class="embedded-example"><a href="/docs/examples/patterns/navigation/subnav" class="js-example">
 View example of the sub-navigation pattern
 </a></div>
 
+<span class="p-label--deprecated">Deprecated</span>
+
+`.p-subnav` is now deprecated, it and its associated child elements can be updated simply by substituting the following class names:
+
+| Deprecated classes | Replaced by                            |
+| ------------------ | -------------------------------------- |
+| `.p-subnav`        | `.p-navigation__item--dropdown-toggle` |
+| `.p-subnav__items` | `.p-navigation__dropdown`              |
+| `.p-subnav__items` | `.p-navigation__dropdown--right`       |
```

Comment:
Should there be `--right` on the deprecated side as well?

Reply:
Yes! Copy paste error - updated